n2v/methods/mrks.py

```diff
@@ -94,7 +94,7 @@ def _vxc_hole_quadrature(self, grid_info=None, atol=1e-5, atol1=1e-4):
         # First loop over the outer set of blocks
         num_block_ten_percent = int(nblocks / 10)
         w1_old = 0
-        print("vxchole quadrature double integral starts (%i points): " % (self.npoints_DFT), end="")
+        print("vxchole quadrature double integral starts (%i points): " % (self.Vpot.grid().npoints()), end="")
```

Comment:
After python 37 they came up with a super easier way of print formatting. Just adding an `f` before the quotation marks and adding the variable inside curly brackets. I think it would be nice to stick to this way since its way much more readable.  For example:
`print( f"vxchole quadrature starts. Using { self.Vpot.grid().npoints():d } number of points" )`
Where the colon is placed after the variable to set a format. `:x.yf` for float... etc

Reply:
Will do.